api/bibliotheca.py

```diff
@@ -98,7 +99,20 @@ class BibliothecaAPI(BaseCirculationAPI, HasSelfTests):
     ] + BaseCirculationAPI.SETTINGS
 
     LIBRARY_SETTINGS = BaseCirculationAPI.LIBRARY_SETTINGS + [
-        BaseCirculationAPI.DEFAULT_LOAN_DURATION_SETTING
+        BaseCirculationAPI.DEFAULT_LOAN_DURATION_SETTING,
+        {
+            "key": ExternalIntegration.DISPLAY_RESERVES,
```

Comment:
The semantics of this are completely upside down, DISPLAY_RESERVES is going to be NULL for most collections and the default behaviour is to always display reserves. We only act if it is set as "no".
So a DISPLAY_RESERVES=NULL value means we should be displaying reserves, this is because of the requirements per the ticket

Reply:
You could use the distinction between None (db NULL) and False to handle this properly.
```
some_var: Optional[bool] = ...

if some_var is None:
    """do default behavior"""
elif some_var is False:
    """do false behavior"""
else:
   """do true behavior"""
```

Reply:
Yes the query only looks for the "no" value at the moment so the semantics work fine, it's just a weird semantic that None and True are equal as opposed to None and False